Library/ViewModels/CommentsViewModelTests.swift

```diff
@@ -831,6 +835,114 @@ internal final class CommentsViewModelTests: TestCase {
     }
   }
 
+  func testConfigureFooterViewWithState_HiddenOnViewDidLoad() {
+    self.configureFooterViewWithState.assertDidNotEmitValue()
+
+    let envelope = CommentsEnvelope.singleCommentTemplate
+    let project = Project.template
```

Comment:
Do we need this constant for this and the other tests if we are only using it in one location (configureWith) ?